src/views/tutorial-view/index.tsx

```diff
@@ -239,6 +234,10 @@ function TutorialView({
 		<>
 			<Head>
 				<link rel="canonical" href={canonicalUrl.toString()} key="canonical" />
+				{/** Don't index non canonical tutorials */}
+				{canonicalUrl.pathname === currentPath ? (
```

Comment:
I think this might be inverted - I'm seeing `noindex` in the AWS collection context (the default), but _not_ seeing it in the Azure context.

```suggestion
				{canonicalUrl.pathname !== currentPath ? (
```

Reply:
:hand-in-face: thank you 🙇 late night PR